clusterloader2/pkg/measurement/util/gatherers/container_resource_gatherer.go

```diff
@@ -81,7 +81,7 @@ func isDaemonPod(pod *corev1.Pod) bool {
 }
 
 // NewResourceUsageGatherer creates new instance of ContainerResourceGatherer
-func NewResourceUsageGatherer(c clientset.Interface, host string, port int, provider provider.Provider, options ResourceGathererOptions, namespace string) (*ContainerResourceGatherer, error) {
+func NewResourceUsageGatherer(c clientset.Interface, host string, port int, provider provider.Provider, options ResourceGathererOptions, namespace string, podLabelSelector ...string) (*ContainerResourceGatherer, error) {
```

Comment:
This would likely affect other test using this measurement, can we make this into a separate change?